main.cpp

```diff
@@ -90,6 +90,7 @@ int flash(po::variables_map &vm, po::parsed_options &parsed)
             ("firmware-file,i", po::value<string>(), "firmware file")
             ("enter-bsl", po::value<bool>()->default_value(true), "enter BSL mode via GPIOs (default: true)")
             ("verbose", po::value<int>()->default_value(0), "verbosity level 0-3 (default: 0)")
+            ("force", po::value<bool>()->default_value(false), "Force the update (default: false)")
```

Comment:
@rckstrh Not sure if what we should make here default. This changes the current behavior so for compatibility reasons we can also make this default to true (= old behavior)

Reply:
I think that while breaking current compatibility it is still a good idea to have the fast path active by default.